src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CLaSPTelemetryService.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using Microsoft.AspNetCore.Razor.Telemetry;
+using Microsoft.CommonLanguageServerProtocol.Framework;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer;
+
+internal class CLaSPTelemetryService(ITelemetryReporter telemetryReporter) : AbstractTelemetryService
```

Comment:
I guess, consider marking this sealed too. :smile:

Reply:
:)